src/server/src/main/java/com/spotify/reaper/resources/view/NodesStatus.java

```diff
@@ -28,34 +29,53 @@
 public class NodesStatus {
   @JsonProperty
   public final List<GossipInfo> endpointStates;
+  private static List<Pattern> endpointNamePatterns = Lists.newArrayList();
+  private static List<Pattern> endpointStatusPatterns = Lists.newArrayList();
+  private static List<Pattern> endpointDcPatterns = Lists.newArrayList();
+  private static List<Pattern> endpointRackPatterns = Lists.newArrayList();
+  private static List<Pattern> endpointLoadPatterns = Lists.newArrayList();
+  private static List<Pattern> endpointReleasePatterns = Lists.newArrayList();
+  private static List<Pattern> endpointSeverityPatterns = Lists.newArrayList();
+  private static List<Pattern> endpointHostIdPatterns = Lists.newArrayList();
+  private static List<Pattern> endpointTokensPatterns = Lists.newArrayList();
```

Comment:
nit-pick: all these `private static` could be `private static final`. That would also means the names should be uppercase.